src/components/Form.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
 import { addBook } from '../redux/books/books';
 
 const Form = () => {
```

Comment:
- [ ] The form also should include an input for an Author to be displayed as you did with the title.

Reply:
Hello @eri8-9 thanks for the review, so for your suggestion to ass the input for an Author it's a good idea but in the origin design there is no input for the Author and they told us to reproduce the same design. I can add the input for the Author later.
Thanks :pray:

Reply:
As requested by the previous code reviewer, please ensure that your New Book React Component has a local state with title, author according to the project requirement.

I believe this was a requirement in a previous milestone:
![image](https://user-images.githubusercontent.com/10332499/160985714-2a015e3c-0de8-45f0-ba94-31effe7404b4.png)

Therefore, I highly suggest you implement the author's input, as you will need it, and will make your life easier on the next milestones.

Explanation
The design that you can see in the sneak peek, is just for the styling part of the project.

Hints
You could shorten the "Book title" input, and in that space insert your "Author" input, next to it, and you will preserve your styling. Like this:
![image](https://user-images.githubusercontent.com/10332499/160985804-12318317-8356-44a7-b016-bef087a9f3bd.png)

I hope this helps.